python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -134,6 +145,11 @@ class ProcessBrightStarsConfig(pipeBase.PipelineTaskConfig,
             " annular flux.",
         default=('BAD', 'CR', 'CROSSTALK', 'EDGE', 'NO_DATA', 'SAT', 'SUSPECT', 'UNMASKEDNAN')
     )
+    doApplySkyCorr = pexConfig.Field(
+        dtype=bool,
+        doc="Apply sky correction before extracting stars?",
```

Comment:
Can you be a bit more specific about which "sky correction" you are referring to here (even just adding "full focal plane" would help...there...are...just...so...many...!!!)?

Reply:
Your update snuck into the bug fix commit 😉